src/util/fork_utils.js

```diff
@@ -15,8 +16,7 @@ const cluster = /** @type {import('node:cluster').Cluster} */ (
  * @param {number?} count number of workers to start.
  * @returns {boolean} true if workers were started.
  */
-function start_workers(count = 0) {
-
+function start_workers(metrics_port, count = 0) {
```

Comment:
can you please replace the order of metrics_port and count, so that count will be first?

Reply:
lint is throwing an error when I try to change the position, `parameter with default value should be last`

Reply:
I think we can remove the default value. WDYT?